erpnext/stock/doctype/batch/batch.json

```diff
@@ -134,9 +135,8 @@
   },
   {
    "fieldname": "supplier",
-   "fieldtype": "Link",
+   "fieldtype": "Data",
```

Comment:
@MyuddinKhatri did you change Supplier to a data field ?

Reply:
Yes